pkg/sql/logictest/tests/cockroach-go-testserver-22.1-22.2/cockroach-go-testserver-22_1-22_2_test: TestLogic_testserver_test_22_1_22_2 failed

pkg/sql/logictest/tests/cockroach-go-testserver-22.1-22.2/cockroach-go-testserver-22_1-22_2_test.TestLogic_testserver_test_22_1_22_2 [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7714752?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7714752?buildTab=artifacts#/) on master @ [1a6e9f885baa124d5ff2996adb966ea15a1a9b2b](https://github.com/cockroachdb/cockroach/commits/1a6e9f885baa124d5ff2996adb966ea15a1a9b2b):


```
=== RUN   TestLogic_testserver_test_22_1_22_2
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/a0cd2ed7268f1ad03abb8f1d57f5fcc9/logTestLogic_testserver_test_22_1_22_23501888860
    test_log_scope.go:79: use -show-logs to present logs inline
    logic.go:3036: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2677/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-22.1-22.2/cockroach-go-testserver-22_1-22_2_test_/cockroach-go-testserver-22_1-22_2_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/testserver_test_22.1_22.2:13: error while processing
    logic.go:3036: init did not finish for node 1
[10:40:53] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2677/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/cockroach-go-testserver-22.1-22.2/cockroach-go-testserver-22_1-22_2_test_/cockroach-go-testserver-22_1-22_2_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/testserver_test_22.1_22.2 with config cockroach-go-testserver-22.1-22.2: 2 tests, 0 failures
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/a0cd2ed7268f1ad03abb8f1d57f5fcc9/logTestLogic_testserver_test_22_1_22_23501888860
--- FAIL: TestLogic_testserver_test_22_1_22_2 (69.69s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_testserver_test_22_1_22_2.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/logictest/logic.go
+++ b/pkg/sql/logictest/logic.go
@@ -1809,6 +1809,7 @@ func (t *logicTest) setup(
 	t.testCleanupFuncs = append(t.testCleanupFuncs, tempExternalIODirCleanup)
 
 	if cfg.UseCockroachGoTestserver {
+		skip.WithIssue(t.t(), 92637)
 		if !bazel.BuiltWithBazel() {
 			skip.IgnoreLint(t.t(), "cockroach-go/testserver can only be uzed in bazel builds")
 		}

